tests/test_overrides_parser.py

```diff
@@ -993,7 +1000,11 @@ def test_override_get_value_element_method(
         param("key={123id: 0}", {"123id": 0}, id="dict_key_int_plus_id"),
         param("key={' abc ': 0}", {" abc ": 0}, id="dict_key_quoted_single"),
         param('key={" abc ": 0}', {" abc ": 0}, id="dict_key_quoted_double"),
-        param("key={a/-\\+.$%*@: 0}", {"a/-\\+.$%*@": 0}, id="dict_key_noquote"),
+        param(
+            "key={a%s: 0}" % UNQUOTED_SPECIAL,
+            {f"a{UNQUOTED_SPECIAL}": 0},
+            id="dict_key_noquote",
+        ),
```

Comment:
I think the leading a is accidental here.
not sure what value it's adding here. I vote to remove it (and in that case, is this test case already covered?)

Reply:
I removed it in 4f89e3889 (also rebased on master to fix a conflict)

There is a test that is very similar in `test_dict_container()` but it doesn't test the same parser rule and doesn't use the `.value()` method (which seems to be the purpose here).